src/list.tsx

```diff
@@ -25,16 +25,21 @@ import {getPrefixedDataAttributes} from './utils/dom';
 import type {DataAttributes, TrackingEvent} from './utils/types';
 
 const useStyles = createUseStyles(({colors, mq}) => ({
+    disabled: {
+        opacity: 0.5,
+        cursor: 'not-allowed',
```

Comment:
I think it is strange (and a bit aggressive) to set a not-allowed cursor to disabled elements

Reply:
this is by spec. Wdyt @AnaMontes11, @yceballost?

Reply:
I think it is a little bit aggressive, but imo, help more than disturb. Let us talk with design team to give an answer

Reply:
I talked with the team, we are going to remove "not allowed" cursor. Disabled states is something to avoid and it is no relevant.

I'm going to update specs!

Reply:
ok, removed